llm/shim_ext_server.go

```diff
@@ -72,7 +72,7 @@ func (llm *shimExtServer) llama_server_release_json_resp(json_resp **C.char) {
 	C.dynamic_shim_llama_server_release_json_resp(llm.s, json_resp)
 }
 
-func newDynamicShimExtServer(library, model string, adapters, projectors []string, numLayers int64, opts api.Options) (extServer, error) {
+func newDynamicShimExtServer(library, model string, adapters, projectors []string, opts api.Options) (extServer, error) {
```

Comment:
Changing the function signature here but not in llm/ext_server_windows.go broke Windows builds.

Reply:
@MatthewCash sorry about that! Should be fixed on `main`

Reply:
Thank you! 😄